### DISCUSSION TITLE
Is there a description of the Json layout and have the keys changed?

### DISCUSSION THREAD

**Discussion by WatcherOfTheSkies** [esnet/iperf#1280] [Q&A] [answered]
Hi,

Is there a description of the Json output available? If so, where is it please?

Has the schema changed between 3.7 and 3.11?

Thank you!

**Comment by bmah888** [comment, marked as answer]
There isn't really any formal schema of the JSON output other than the code. There have been a few changes to the JSON format (I hesitate to call it an actual schema) between 3.7 and 3.11, they might be mentioned in the release notes or the commit logs. We try to preserve compatibility where possible between different versions of iperf3, for example not gratuitously renaming fields, etc. In general, code that can parse the JSON output from iperf-3.7 should be able to understand the output from iperf-3.11.
